src/rgw/rgw_admin.cc

```diff
@@ -2993,6 +2993,9 @@ int main(int argc, char **argv)
 	  cerr << "realm.rename failed: " << cpp_strerror(-ret) << std::endl;
 	  return -ret;
 	}
+        cout << "Realm name updated. Note that this change only applies to "
+            "the current cluster, so this command must be run separately "
```

Comment:
@amitkumar50 Backports are cherry-picked from master, so any changes would have to go into master, first.

("this command must be run separately" is fine as it is, though)